dstest/dstest_test.go

```diff
@@ -115,4 +115,9 @@ raleigh,250000,50.65,true
 	if !bytes.Equal(mfdata, data) {
 		t.Errorf("memfile data mismatch")
 	}
+
+	rendered, err := tc.RenderedFile()
+	if err != nil {
+		t.Errorf("reading %s: %s", RenderedFilename, rendered)
```

Comment:
No guarantee that `rendered` is valid if `err` was returned. Should using `err` instead:

`t.Errorf("reading %s: %s", RenderedFilename, err)`